ENIGMAsystem/SHELL/Universal_System/roomsystem.cpp

```diff
@@ -268,6 +245,32 @@ namespace enigma
 
     instance_event_iterator = new inst_iter(NULL,NULL,NULL);
 
+    // Initialize view variants so they do not throw uninitialized variable access errors.
```

Comment:
Why does moving this block down effect the behavior you're after? Where are the events that read these variables actually called?

Reply:
If you're going to call `follow_object` the object needs to be created, right?

Reply:
I see. GitHub elided the create loop. I assumed it was below with the call loop.